src/components/molecules/TablesUserList/TablesUserList.tsx

```diff
@@ -36,6 +36,7 @@ export interface TablesUserListProps {
   leaveText?: string;
   venue: WithId<AnyVenue>;
   venueId: string;
+  type: VenueTemplate;
```

Comment:
shouldn't `template` be the more appropriate name since that's what we're using thorough the code base?

Reply:
yep, replaced 2ba4481